fragments/functions.sh

```diff
@@ -264,11 +264,11 @@ getAppVersion() {
 #            targetDir="/Applications/Utilities"
 #        fi
     else
-    #    applist=$(mdfind "kind:application $appName" -0 )
         printlog "name: $name, appName: $appName"
-        applist=$(mdfind "kind:application AND name:$name" -0 )
+        # mdfind now handling if kind is either Application or App
+        applist=$(mdfind 'kMDItemContentType:com.apple.application AND kMDItemFSName:"'"$name"'"' -0 2>/dev/null)
```

Comment:
not a fan of your quotations here. They're not wrong per se, but entirely unreadable. Maybe use escaped quotes instead:

```
applist=$(mdfind "kMDItemContentType:com.apple.application AND kMDItemFSName:\"$name\"" -0 2>/dev/null)
```

Reply:
Updated!